server/service/src/login.rs

```diff
@@ -152,6 +157,13 @@ impl LoginService {
                         LoginError::InternalError("Failed to read credentials".to_string())
                     }
                     VerifyPasswordError::DatabaseError(e) => LoginError::DatabaseError(e),
+                    VerifyPasswordError::EmptyHashedPassword => {
+                        if connection_failure {
+                            LoginError::MSupplyCentralNotReached
+                        } else {
+                            LoginError::InternalError("Corrupted credentials".to_string())
```

Comment:
Changed this error message, as it's currently mapping all the way through to the frontend, and this felt a little too raw:

![Screenshot 2024-09-04 at 4 11 28 PM](https://github.com/user-attachments/assets/025db2a8-88d4-45ae-95c3-0fc858a3d43d)

Arguably we shouldn't display the backend internal error messages to the user, but probably more helpful than not at the moment 😅